### PR TITLE
Cortx 33386

### PR DESCRIPTION
**Pull request by kachhwahadivya**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by ravindrachoudhari — APPROVED** [review]

**Comment by kachhwahadivya** [comment]
[collection.log](https://github.com/Seagate/cortx-test/files/9357870/collection.log)

File: libs/csm/rest/csm_rest_alert.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 from commons import errorcodes as err
 from commons.exceptions import CTException
 from libs.csm.rest.csm_rest_test_lib import RestTestLib
-
+# pylint: disable-msg=unexpected-keyword-arg
```
Comment (sonalk0209) [inline libs/csm/rest/csm_rest_alert.py:33]: why blank line here?

File: commons/constants.py [hunk 2]
```diff
@@ -209,9 +209,7 @@ class Rest:
     S3_ACCOUNTS = "s3_accounts"
     ACC_NAME = "account_name"
     ACC_EMAIL = "account_email"
-    SECRET_KEY = "secret_key"
```
Comment (sonalk0209) [inline commons/constants.py:212]: did we check if these are not getting used by anyone else other than CSM files?

File: libs/csm/rest/csm_rest_bucket.py [hunk 3]
```diff
@@ -26,6 +26,8 @@
 from commons.exceptions import CTException
 from commons.utils import config_utils
 from libs.csm.rest.csm_rest_test_lib import RestTestLib
+# pylint: disable-msg=unexpected-keyword-arg
+
```
Comment (sonalk0209) [inline libs/csm/rest/csm_rest_bucket.py:30]: check for blank lines, if not needed we can remove

**Review by sonalk0209 — APPROVED** [review]
Minor comments, please check @kachhwahadivya

File: commons/constants.py (hunk 2 above)
Comment (kachhwahadivya) [reply commons/constants.py:212]: yes, they are not getting except the place it is changed

File: libs/csm/rest/csm_rest_bucket.py (hunk 3 above)
Comment (kachhwahadivya) [reply libs/csm/rest/csm_rest_bucket.py:30]: Added a blank line as initially there was no separation between class and imports.

File: libs/csm/rest/csm_rest_alert.py (hunk 1 above)
Comment (kachhwahadivya) [reply libs/csm/rest/csm_rest_alert.py:33]: same as above

**Pull request merged**